### PR TITLE
css gradient added

### PR DESCRIPTION
**Pull request by blakeworsley**

### CODE REVIEW

File: app/css/index.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
+.rgba-copied::after, .hex-copied::after {
+  content: 'copied to clipboard';
+  background: white;
+  border: 3px solid #777;
+  padding: 5px;
+}
```
Comment (kylem038) [inline app/css/index.css:6]: What is this doing? Is this changing the style of the button once it's clicked?

File: app/main.js [hunk 2]
```diff
@@ -4,10 +4,12 @@ let mainWindow = null;
...
-    width: 300,
-    height: 500,
-    maxWidth: 800,
-    maxHeight: 600
+    width: 1000,
+    height: 600,
+    // width: 300,
+    // height: 500,
+    // maxWidth: 800,
+    // maxHeight: 600
```
Comment (kylem038) [inline app/main.js:12]: Why are we getting rid of the maxWidth / maxHeight?

Comment (blakeworsley) [reply app/main.js:12]: This is a temp fix in order to work in full screen when we are in developer mode.

File: app/css/index.css (hunk 1 above)
Comment (blakeworsley) [reply app/css/index.css:6]: This is the copy to clipboard message styling.

Comment (kylem038) [reply app/css/index.css:6]: Nice!

File: app/main.js (hunk 2 above)
Comment (kylem038) [reply app/main.js:12]: Cool - thanks for the explanation.

**Pull request merged**